inception-project/inception
# #1934 - Convert activity dashlet to vue component

**What's in the PR**
* New vue-based activity dashlet for the project dashboard page

**How to test manually**
* Open a project
* Annotate/curate some stuff
* Check the activity in the dashlet
* Note: re-opening a curated document via the dashlet currently does not work, but that seems to be a fault of the curation page, not of the dashlet. The curation page does open, but it does not open the requested document.

**Automatic testing**

**Documentation**
